lambda-buffers-proto/compiler.proto

```diff
@@ -463,19 +463,16 @@ message TyClassRef {
   }
 }
 
-/* Type class instances (rules)
+/* Type class instance deriving clauses
 
-Instance clauses enable users to specify 'semantic' rules for their types.
+Deriving clauses enable users to specify 'semantic' rules for their types.
 */
-message InstanceClause {
+message DeriveClause {
```

Comment:
wording nit; Initially I used "instance CLAUSE" to denote that it's a "clause" in a rule set (which is a set of clauses). I guess it's a pretty general term, can be applied in many settings but perhaps we could just call it `Derive` instead?

Reply:
I think `DeriveClause` is OK. Maybe `DerivingClause`? I don't think "clause" has a single fixed technical meaning in this context.